packit/config/aliases.py

```diff
@@ -62,7 +61,11 @@ def get_aliases() -> dict[str, list[Distro]]:
 
     try:
         opensuse_aliases = opensuse_distro_aliases.get_distro_aliases()
-    except requests.RequestException:
+    except Exception as e:
```

Comment:
Can't we catch just the `IndexError` here? Otherwise LGTM, thanks!

Reply:
We can, but the point of this is to make sure `get_aliases()` always works, be it with fresh or cached aliases, even if there is a different bug (and a different exception raised) in the implementation.

Reply:
Perhaps I could change the `warning()` to `error()` so it shows up in Sentry? WDYT?